editor/editor_help_search.cpp

```diff
@@ -766,9 +793,26 @@ String EditorHelpSearch::Runner::_build_method_tooltip(const DocData::ClassDoc *
 		}
 	}
 	tooltip += ")";
+	tooltip += _build_keywords_tooltip(p_doc->keywords);
 	return tooltip;
 }
 
+String EditorHelpSearch::Runner::_build_keywords_tooltip(const String &p_keywords) const {
+	String tooltip;
+	if (p_keywords.is_empty()) {
+		return tooltip;
+	}
+
+	tooltip = "\n    " + TTR("Search Keywords") + ": ";
+
+	for (const String &keyword : p_keywords.split(",")) {
+		tooltip += "\"" + keyword.strip_edges() + "\", ";
```

Comment:
There's a `quote()` for Strings. I don't think stripping the edges is necessary here. Keywords should not contain leading or trailing spaces.
```suggestion
		tooltip += keyword.quote() + ", ";
```
This loop part is a bit hacky but I do not have suggestions right now. If it works, it works.

Reply:
The keywords come directly from the XML attribute `keywords="..."` and there may be spaces after each comma.
Would need to remove the spaces when loading the keywords string in `doc_data.h` but that's not currently done.

Reply:
I see, in that case `strip_edges()` could be there for good measure.